shared/js/content-scripts/content-scope-messaging.js

```diff
@@ -0,0 +1,52 @@
+function getSecretElement () {
+    // Main-world script ran first, grab the element.
+    if (document.getElementById('ddg-secret')) {
```

Comment:
I think we should consider trying the following approaches instead:

1. window.sessionStorage and addEventListener('storage')
2. Firing a custom event and storing the data on window.navigator.duckduckgo

Both I think will have less side effects to the page, if not we should probably use an inert tag that's never visible; like <script> but with a type of application/json or something.

Reply:
_(For reference, Jonathan and I discussed this in MatterMost. Since we figured out how to ensure the isolated world script was executed first, it was enough to simply emit the secret event from the main world script.)_